### PR TITLE
Internal db

### PR DESCRIPTION
**Pull request by magnusihle**
Added Express backend for sending MongoDB information to frontend.

### CODE REVIEW

File: client/src/components/EmployeTable.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import ChevronLeft from '@material-ui/icons/ChevronLeft'
+import ChevronRight from '@material-ui/icons/ChevronRight'
+import FirstPage from '@material-ui/icons/FirstPage'
+import LastPage from '@material-ui/icons/LastPage'
+import Add from '@material-ui/icons/Add'
+import Check from '@material-ui/icons/Check'
+import FilterList from '@material-ui/icons/FilterList'
+import Remove from '@material-ui/icons/Remove'
+
+const styles = theme => ({
```
Comment (zhenik) [inline client/src/components/EmployeTable.js]: mixing mvp, styles should be in separate file

File: client/src/components/forms/onboard/startOnboard.js [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+    const [isFalse, setIsFalse] = useState(false);
+  function handleClick(e) {
+    e.preventDefault();
+
+    let data = {
+      firstName: firstName,
+      lastName: lastName
+    };
+
+    fetch("/api/v1/checkEmployes", {
```
Comment (zhenik) [inline client/src/components/forms/onboard/startOnboard.js]: I would suggest have this URL from config

File: worker/src/index.js [hunk 3]
```diff
@@ -1,10 +1,43 @@
...
+//  Testing Node js as a backend
+const express = require('express');
+
 
 //  Move data to DB
+mongoCreateDB();
+mongoCreateCollection();
+const app = express();
+
+app.get('/api/v1/getEmployes', (req, res, next) => {
```
Comment (zhenik) [inline worker/src/index.js]: `/api/v1/` - prefix could be in config

File: client/src/components/EmployeTable.js (hunk 1 above)
Comment (magnusihle) [reply client/src/components/EmployeTable.js]: Okey, should I do that even tho they use it this way in the documentation? 

Material UI [Sylles API](https://material-ui.com/styles/api/)

File: worker/src/index.js (hunk 3 above)
Comment (magnusihle) [reply worker/src/index.js]: That's smart :)

File: client/src/components/EmployeTable.js (hunk 1 above)
Comment (zhenik) [reply client/src/components/EmployeTable.js]: I would say that in docs it is done for example only.

**Pull request merged**